docs/resources/vault_cluster.md

```diff
@@ -121,3 +121,10 @@ Import is supported using the following syntax:
 # The import ID is {cluster_id}
 terraform import hcp_vault_cluster.example vault-cluster
 ```
+
+## Tutorials
+
+Refer to the following tutorials for additional usage examples:
+
+- [Deploy HCP Vault with Terraform](https://learn.hashicorp.com/tutorials/cloud/terraform-hcp-provider-vault) 
+- [Deploy HCP Vault Performance Replication with Terraform](https://learn.hashicorp.com/tutorials/cloud/vault-replication-terraform)
```

Comment:
To get the "Check Generated Docs" CI step to pass, try making this change in the file `templates/resources/vault_cluster.md.tmpl` and running `make gencheck`.